shared/chat/conversation/messages/wallet-payment/index.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import * as React from 'react'
+import {Box2, Button, Divider, Icon, Markdown, Text} from '../../../../common-adapters'
+import type {IconType} from '../../../../common-adapters/icon.constants'
+import {globalColors, styleSheetCreate} from '../../../../styles'
+
+export type Props = {
+  action: string,
+  amount: string,
+  balanceChange: string,
+  balanceChangeColor: string,
+  icon: IconType,
+  memo: string,
+  onSend?: () => void,
+  pending: boolean,
+  sendButtonLabel?: string,
+}
+
+export default (props: Props) => {
+  return (
+    <Box2 direction="vertical" gap="xtiny" fullWidth={true}>
+      <Box2 direction="horizontal" fullWidth={true} style={styles.headingContainer}>
+        <Box2 direction="horizontal" gap="xtiny" style={styles.headingContainer}>
+          <Icon type={props.icon} color={globalColors.purple2} fontSize={12} />
+          <Text type="BodySmall" style={styles.purple}>
+            {props.action}{' '}
+            <Text type="BodySmallExtrabold" style={styles.purple}>
+              {props.amount}
+            </Text>
+            {props.pending ? '...' : '.'}
+          </Text>
+        </Box2>
+        {!!props.balanceChange && (
+          <Box2 direction="horizontal">
+            <Text type="BodyExtrabold" style={{color: props.balanceChangeColor}}>
+              {props.balanceChange}
+            </Text>
+          </Box2>
+        )}
+      </Box2>
+      <Box2 direction="horizontal" gap="small" fullWidth={true}>
+        <Divider vertical={true} style={styles.quoteMarker} />
```

Comment:
Moving the quote marker out here is to keep some nice features of `Markdown`, like emoji-only strings being bigger.